### ISSUE TITLE
pip install failed on win10

### ISSUE THREAD

**Issue by Robird** [aubio/aubio#264]
```
  Info: libaubio was not installed or built locally with waf, adding src/
  workaround on win64 to avoid empty pre-processor output
  Running command: D:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\\cl.exe -E -DAUBIO_UNSTABLE=1 -Isrc src\aubio.h
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "C:\Users\Admin\AppData\Local\Temp\pip-install-urdlf2mu\aubio\setup.py", line 95, in <module>
      'aubiocut = aubio.cut:main',
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\site-packages\setuptools\__init__.py", line 145, in setup
      return distutils.core.setup(**attrs)
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\core.py", line 148, in setup
      dist.run_commands()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\dist.py", line 955, in run_commands
      self.run_command(cmd)
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\dist.py", line 974, in run_command
      cmd_obj.run()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\site-packages\wheel\bdist_wheel.py", line 192, in run
      self.run_command('build')
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\dist.py", line 974, in run_command
      cmd_obj.run()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\command\build.py", line 135, in run
      self.run_command(cmd_name)
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\dist.py", line 974, in run_command
      cmd_obj.run()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\command\build_ext.py", line 339, in run
      self.build_extensions()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\command\build_ext.py", line 448, in build_extensions
      self._build_extensions_serial()
    File "c:\local\anaconda3-4.3.1-windows-x86_64\envs\tfjs\lib\distutils\command\build_ext.py", line 473, in _build_extensions_serial
      self.build_extension(ext)
    File "python\lib\moresetuptools.py", line 192, in build_extension
      usedouble=enable_double)
    File "python\lib\gen_external.py", line 271, in generate_external
      c_declarations = get_c_declarations(header, usedouble=usedouble)
    File "python\lib\gen_external.py", line 101, in get_c_declarations
      cpp_output = get_cpp_output(header=header, usedouble=usedouble)
    File "python\lib\gen_external.py", line 129, in get_cpp_output
      % err_output.decode('utf8'))
  UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 0: invalid continuation byte
  ----------------------------------------
  ERROR: Failed building wheel for aubio
  Running setup.py clean for aubio
Failed to build aubio
```

**Comment by piem** [comment]
hello @Robird 

what locale are you using? something like [this](https://superuser.com/a/670200) might help finding out.

thanks

**Comment by Nymerod** [comment]
Hi, I didn't know whether I should submit a new issue or not but I think its pretty similar ?


<details> <summary> error log </summary>

C:\Users\Sora\Documents\Code\aubio\aubio-0.4.6-win64>pip install aubio
Collecting aubio
  Using cached https://files.pythonhosted.org/packages/cd/80/302d89240603e5347c7f8026c8b02c59f8dfaec66c91a743d82de7c86006/aubio-0.4.9.tar.gz
Requirement already satisfied: numpy in c:\users\sora\appdata\local\programs\python\python36\lib\site-packages (from aubio) (1.17.4)
Installing collected packages: aubio
    Running setup.py install for aubio ... error
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\sora\appdata\local\programs\python\python36\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Sora\\AppData\\Local\\Temp\\pip-install-rl42e0vw\\aubio\\setup.py'"'"'; __file__='"'"'C:\\Users\\Sora\\AppData\\Local\\Temp\\pip-install-rl42e0vw\\aubio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Sora\AppData\Local\Temp\pip-record-2a223vrx\install-record.txt' --single-version-externally-managed --compile
         cwd: C:\Users\Sora\AppData\Local\Temp\pip-install-rl42e0vw\aubio\
    Complete output (73 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\aubio
    copying python\lib\aubio\cmd.py -> build\lib.win-amd64-3.6\aubio
    copying python\lib\aubio\cut.py -> build\lib.win-amd64-3.6\aubio
    copying python\lib\aubio\midiconv.py -> build\lib.win-amd64-3.6\aubio
    copying python\lib\aubio\slicing.py -> build\lib.win-amd64-3.6\aubio
    copying python\lib\aubio\__init__.py -> build\lib.win-amd64-3.6\aubio
    running build_ext
    checking for aubio = 0.4.9
    Running "pkg-config --libs --cflags aubio = 0.4.9" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    Info: aubio 0.4.9 was not found by pkg-config
    Info: looking for *optional* additional packages
    checking for libavcodec
    Running "pkg-config --libs --cflags libavcodec" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for libavformat
    Running "pkg-config --libs --cflags libavformat" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for libavutil
    Running "pkg-config --libs --cflags libavutil" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for libswresample
    Running "pkg-config --libs --cflags libswresample" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for libavresample
    Running "pkg-config --libs --cflags libavresample" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for sndfile
    Running "pkg-config --libs --cflags sndfile" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    checking for samplerate
    Running "pkg-config --libs --cflags samplerate" failed: FileNotFoundError(2, 'Le fichier spécifié est introuvable', None, 2, None)
    Info: libaubio was not installed or built locally with waf, adding src/
    Running command: C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.23.28105\bin\HostX86\x64\cl.exe -E -DAUBIO_UNSTABLE=1 -Isrc src\aubio.h
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Sora\AppData\Local\Temp\pip-install-rl42e0vw\aubio\setup.py", line 95, in <module>
        'aubiocut = aubio.cut:main',
      File "c:\users\sora\appdata\local\programs\python\python36\lib\site-packages\setuptools\__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\site-packages\setuptools\command\install.py", line 61, in run
        return orig.install.run(self)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\command\install.py", line 545, in run
        self.run_command('build')
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\command\build.py", line 135, in run
        self.run_command(cmd_name)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\command\build_ext.py", line 339, in run
        self.build_extensions()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\command\build_ext.py", line 448, in build_extensions
        self._build_extensions_serial()
      File "c:\users\sora\appdata\local\programs\python\python36\lib\distutils\command\build_ext.py", line 473, in _build_extensions_serial
        self.build_extension(ext)
      File "python\lib\moresetuptools.py", line 192, in build_extension
        usedouble=enable_double)
      File "python\lib\gen_external.py", line 271, in generate_external
        c_declarations = get_c_declarations(header, usedouble=usedouble)
      File "python\lib\gen_external.py", line 101, in get_c_declarations
        cpp_output = get_cpp_output(header=header, usedouble=usedouble)
      File "python\lib\gen_external.py", line 129, in get_cpp_output
        % err_output.decode('utf8'))
    UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 54: invalid start byte
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\sora\appdata\local\programs\python\python36\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Sora\\AppData\\Local\\Temp\\pip-install-rl42e0vw\\aubio\\setup.py'"'"'; __file__='"'"'C:\\Users\\Sora\\AppData\\Local\\Temp\\pip-install-rl42e0vw\\aubio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Sora\AppData\Local\Temp\pip-record-2a223vrx\install-record.txt' --single-version-externally-managed --compile Check the logs for full command output.'


</details>

After checking out my locale like said above I got this 

<details>
<summary> local </summary>
[System.Text.Encoding]::Default

IsSingleByte      : True
BodyName          : iso-8859-1
EncodingName      : Europe de l'Ouest (Windows)
HeaderName        : Windows-1252
WebName           : Windows-1252
WindowsCodePage   : 1252
IsBrowserDisplay  : True
IsBrowserSave     : True
IsMailNewsDisplay : True
IsMailNewsSave    : True
EncoderFallback   : System.Text.InternalEncoderBestFitFallback
DecoderFallback   : System.Text.InternalDecoderBestFitFallback
IsReadOnly        : True
CodePage          : 1252

</details>

Would appreciate any help. =)

**Comment by kewbish** [comment]
I've the same problem. Any resolution as of yet?

**Comment by piem** [comment]
hello @Robird, @Nymerod, and @kewbish 

could you try reproducing this from the latest git?

    $ pip install git+https://git.aubio.org/aubio/aubio/

this seems to be a duplicate of gh-247, which was fixed with 66fff6c.

cheers, piem

**Comment by piem** [comment]
hi all,

i'm closing this issue as a duplicate of gh-247. Please re-open this issue providing more info if you are still having troubles installing from master.

cheers, piem
